wire-ios/Wire-iOS Tests/ConversationMessageCell/MessageActionsViewControllerTests.swift

```diff
@@ -130,7 +133,12 @@ final class MessageActionsViewControllerTests: XCTestCase {
 
 }
 
-final class BasicReactionPickerTests: ZMSnapshotTestCase {
+final class BasicReactionPickerTests: XCTestCase {
```

Comment:
Could we move this tests case to a new file?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/2ea5a6ce08421116e5100dd118f0f06fcc5968ed